static/app/components/replays/breadcrumbs/replayTimelineEvents.tsx

```diff
@@ -114,8 +114,6 @@ function Event({
           setActiveTab('dom');
           break;
         case 'error':
-          setActiveTab('issues');
-          break;
```

Comment:
Was gonna say that I don't like relying on the fallthrough behavior of switch statements because potential of introducing bugs is higher, but we're doing it already for navigation/debug.

Reply:
i looked at merging the two switch statements, but it seemed premature with only 2 examples. it is what it is